python/lsst/pipe/drivers/coaddDriver.py

```diff
@@ -320,7 +321,13 @@ def coadd(self, cache, data):
             with self.logOperation("detection on {}".format(patchRef.dataId),
                                    catch=True):
                 idFactory = self.detectCoaddSources.makeIdFactory(patchRef)
-                expId = int(patchRef.get(self.config.coaddName + "CoaddId"))
+                try:
+                    expId = getGen3CoaddExposureId(
+                        patchRef, coaddName=self.detectCoaddSources.config.coaddName, log=self.log)
+                except Exception as e:
+                    self.log.warn("Setting exposureId to match Gen3 failed with: %s.  Falling back to "
```

Comment:
This log message was really helpful, so please add it to the  try: getGen3CoaddExposureId blocks you have in pipe_tasks and meas_base too. I wish I looked at this repo first.

Reply:
log.warning

Reply:
Sorry again...the `try/except/log.warning` are all now in the  [getGen3CoaddExposureId function](https://github.com/lsst/coadd_utils/pull/30/commits/f09958e309f6e32fd1fff143f57b0c88d231616c#diff-c4961a7427d6736ea2e9052d39925f90d1ef7b2e0c7175fee9cc60c25b89f7bcR79-R85) itself, so removed it here.